pyproject.toml

```
@@ -1,183 +1,185 @@
-vrplib = "^1.4.0"
-tqdm = "^4.64.1"
-tomli = "^2.0.1"
-
-
-[tool.poetry.group.docs]
-optional = true
-
-
-[tool.poetry.group.docs.dependencies]
-sphinx = ">=7.3.0,<8.2.0"
-nbsphinx = ">=0.8.9"
-ipython = ">=8.6.0"
-numpydoc = ">=1.5.0"
-sphinx-immaterial = ">=0.11.12"
-
-
-[tool.poetry.group.examples]
-optional = true
-
-
-[tool.poetry.group.examples.dependencies]
-jupyter = ">=1.0.0"
-tabulate = "^0.9.0"
-
-
-[tool.poetry.group.dev.dependencies]
-pre-commit = "^2.20.0"
-pytest = ">=6.0.0"
-pytest-cov = "^6.1.1"
-pytest-codspeed = ">=2.2.1"
-pytest-xdist = "^3.6.1"
-pytest-sugar = "^1.0.0"
-codecov = "*"
-
-# These are used in the build script: for compiling the library (meson, ninja)
-# and generating docs (docblock) and coverage reports (gcovr).
-meson = "^1.0.0"
-ninja = "^1.11.1"
-gcovr = "^7.2"
-docblock = ">=0.1.6"
-
-
-[tool.poetry.scripts]
-pyvrp = "pyvrp.cli:main"
-
-
-[tool.black]
-line-length = 79
-
-
-[tool.ruff]
-ignore-init-module-imports = true
-line-length = 79
-select = [
-    "E", "F", "I", "NPY", "PYI", "Q", "RET", "RSE", "RUF", "SLF", "SIM", "TCH"
-]
-
-
-[tool.ruff.isort]
-case-sensitive = true
-known-first-party = ["pyvrp", "tests"]
-
-
-[tool.mypy]
-ignore_missing_imports = true
-
-
-[tool.pytest.ini_options]
-addopts = [
-    "--cov",
-    "--cov-report=xml",
-    "--cov-report=term",
-    "--numprocesses=auto",
-]
-testpaths = "tests"
-
-
-[tool.coverage.run]
-omit = [
-    "build_extensions.py",  # build entrypoint
-    "extract_docstrings.py",  # build script
-    "pyvrp/show_versions.py",  # only prints debug information
-    "pyvrp/cli.py",  # tested in other ways than unit tests
-    ".venv/*",
-    "benchmarks/*",
-    "docs/*",
-    "tests/*",
-]
-
-
-[tool.coverage.report]
-exclude_lines = [
-    "pragma: no cover",
-    "@abstract",
-    "if TYPE_CHECKING:",
-]
-
-
-[tool.cibuildwheel]
-# We do not support old Python versions and somewhat uncommon platforms. For
-# musllinux-based builds we assume users can compile the thing themselves.
-build = "cp310-* cp311-* cp312-* cp313-*"
-skip = "pp* *_ppc64le *_i686 *_s390x *-win32 *-musllinux*"
-build-frontend = "build"
-test-requires = [
-    "pytest>=6.0.0",
-    "pytest-cov>=6.1.1",
-    "pytest-xdist>=3.6.1",
-    "pytest-sugar>=1.0.0",
-]
-test-command = "pytest {package}/tests"
-
-
-[tool.poetry.build]
-generate-setup-file = false
-script = "build_extensions.py"
-
-
-[build-system]
-# We need meson and ninja to build the C++ extensions, and docblock to extract
-# documentation for the extensions.
-requires = ["poetry", "meson", "ninja", "docblock"]
-build-backend = "poetry.core.masonry.api"
+[project]
+name = "pyvrp"
+version = "0.12.0a0"
+description = "A state-of-the-art vehicle routing problem solver."
+authors = [
+    { name = "Niels Wouda", email = "nielswouda@gmail.com" },
+    { name = "Leon Lan", email = "leon.lanyidong@gmail.com" },
+    { name = "Wouter Kool", email = "wouter.kool@ortec.com" },
+]
+license = { file = "LICENSE.md" }
```

New format for licenses: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

```suggestion
license = "MIT"
license-files = ["LICENSE.md"]
```